gobblin-aws/src/test/java/org/apache/gobblin/aws/GobblinAWSClusterLauncherTest.java

```diff
@@ -64,6 +44,19 @@
 import org.apache.gobblin.cluster.TestHelper;
 import org.apache.gobblin.cluster.TestShutdownMessageHandlerFactory;
 import org.apache.gobblin.testing.AssertWithBackoff;
+import org.apache.helix.HelixManager;
+import org.apache.helix.HelixManagerFactory;
+import org.apache.helix.InstanceType;
+import org.apache.helix.model.Message;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
```

Comment:
Can we follow the import order on the style guide?
https://gobblin.apache.org/docs/developer-guide/CodingStyle/
There's an intellij file you can set as the guide and intellij can autoformat the imports for you.
On a side note I should look into trying to enforce this as part of checkstyle but that'll come later<sup>tm</sup>